src/fund_transf_module.rs

```diff
@@ -0,0 +1,483 @@
+imports!();
```

Comment:
I think the data inside the contract should be something like this:
user -> list of money
waiting list of money
stake list of money
unstake list of money
money -> user and state ( inactive, stake, unstake, inactive)

We need the defined state machine how can the money change its state - and in case of one change where is the data changed as well.

Reply:
To simplify the code even more, I think we can make the following limitation: money cannot change its owner. It can change only in the following order: can be created as inactive always, moving to waiting, stake, unstake, inactive